### PR TITLE
add me as a participant

### PR DESCRIPTION
**Pull request by lav-joshi**

### CODE REVIEW

File: Data/participants.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
       "facebookLink": "https://www.facebook.com/this.is.harshit.gupta",
       "githubLink": "https://github.com/harshitg00"
     }
```
Comment (dojutsu-user) [inline Data/participants.json]:
```suggestion
    },
```

File: Data/participants.json [hunk 2]
```diff
@@ -14,5 +14,13 @@
...
+    {
+      "name": "Lav Joshi",
+      "email": "lcs2019022@iiitl.ac.in",
+      "profilePicture": "https://avatars.githubusercontent.com/lav-joshi",
+      "facebookLink": "https://www.facebook.com/lav.joshi.923",
+      "githubLink": "https://github.com/lav-joshi"
+    }
+
```
Comment (dojutsu-user) [inline Data/participants.json:24]:
```suggestion
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by dojutsu-user** [comment]
Please don't make PR from your `master` branch. This is not considered as a good practice.
Make a separate branch and then make a PR from it.

**Comment by lav-joshi** [comment]
Done sir

**Comment by dojutsu-user** [comment]
Please resolve the merge conflicts.

**Comment by rishabhshuklax** [comment]
Make a new PR from a new branch. For now, I am closing this one.

**Pull request closed without merging**